.github/workflows/standalone.yml

```diff
@@ -7,87 +7,86 @@ on:
       matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        version: [10, 11, 12]
         include:
-          - os: windows-latest
-            compiler: gnu
-            cc_cmd: gcc
-            fc_cmd: gfortran
-            cxx_cmd: g++
-          - os: ubuntu-latest
-            compiler: gnu
-            cc_cmd: gcc
-            fc_cmd: gfortran
-            cxx_cmd: g++
-          - os: macos-latest
-            compiler: gnu
+          - compiler: gnu
             cc_cmd: gcc
             fc_cmd: gfortran
             cxx_cmd: g++
           - os: windows-latest
             compiler: intel
-            hpckit: https://registrationcenter-download.intel.com/akdlm/irc_nas/19085/w_HPCKit_p_2023.0.0.25931_offline.ex
-            # hpckit: 'https://registrationcenter-download.intel.com/akdlm/irc_nas/xxx/w_HPCKit_p_2022.3.1.19755_offline.exe'
-            script: 'install_windows.bat'
-            cc_cmd: icc
+            version:
+            cc_cmd: icl
             fc_cmd: ifort
-            cxx_cmd: icpc
+            cxx_cmd: icl
           - os: ubuntu-latest
             compiler: intel
-            hpckit: 'https://registrationcenter-download.intel.com/akdlm/irc_nas/18975/l_HPCKit_p_2022.3.1.16997_offline.sh'
-            script: 'install_linux.sh'
+            version:
             cc_cmd: icc
             fc_cmd: ifort
             cxx_cmd: icpc
           - os: macos-latest
             compiler: intel
-            hpckit: 'https://registrationcenter-download.intel.com/akdlm/irc_nas/18977/m_HPCKit_p_2022.3.1.15344_offline.dmg'
-            script: 'install_macos.sh'
+            version:
             cc_cmd: icc
             fc_cmd: ifort
             cxx_cmd: icpc
     runs-on: ${{ matrix.os }}
     steps:
       - name: Check out GALAHAD
         uses: actions/checkout@v3
+
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
           python-version: '3.11'
+
       - name: Install Meson and Ninja
         run: pip install meson ninja
+
+      - name: Set environment variables
+        shell: bash
+        run: |
+          echo "GALAHAD=/home/runner/work/GALAHAD/GALAHAD" >> $GITHUB_ENV
+
       - name: Install GNU C and Fortran compilers
-        uses: modflowpy/install-gfortran-action@v1
+        if: matrix.compiler == 'gnu'
+        uses: awvwgk/setup-fortran@main
+        with:
+          compiler: gcc
+          version: ${{ matrix.version }}
+
       - name: Install Intel OpenAPI C and Fortran compilers
         if: matrix.compiler == 'intel'
-        run: |
-          .github/scripts/${{ matrix.script }} "${{ matrix.hpckit }}" all
-      - name: Setup GALAHAD build
+        uses: modflowpy/install-intelfortran-action@v1
+
+      - name: Setup GALAHAD
         shell: bash
         run: |
-          [[ "${{ matrix.compiler }}" == "intel" ]] && source /opt/intel/oneapi/setvars.sh
-          meson setup builddir -Dciface=true
+          meson setup builddir
```

Comment:
We've lost `-Dciface=true` is that now the default?

Reply:
Yes, it's now the default, like HSL.